kickstarts/partials/packages/includes.ks.erb

```diff
@@ -20,11 +21,8 @@ WALinuxAgent
 dracut-config-generic
 <% when "hyperv" %>
 hyperv-daemons
-<% when "libvirt" %>
-qemu-guest-agent
-<% when "ovirt" %>
+<% when "libvirt", "ovirt" %>
```

Comment:
ovirt goes away now, right?

Reply:
Not yet, there's still a few things that need to be done before we can remove the build.  I'm trying to test a libvirt image in ovirt, but having trouble getting the image uploaded